src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,58 @@
+@use '../utils/variables' as *;
+
+.nav {
+  height: 60px;
+
+  &__list {
+    list-style-type: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    align-items: center;
+    display: flex;
+    position: relative;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $dark-blue;
+    font-style: normal;
+    font-weight: 500;
+    height: 60px;
+    line-height: 60px;
```

Comment:
delete line-height style because on small resolutions it looks like this:
![image](https://github.com/mate-academy/layout_catalog/assets/126968958/9ff2a6ff-dbdb-43d2-889b-42ca9fc2532c)